packages/curvenote-cli/src/session/session.ts

```diff
@@ -246,7 +247,11 @@ export class Session implements ISession {
         });
         if (response.ok) {
           this.log.debug(`Response ok.`);
-          this.$config = (await response.json()) as CLIConfigData;
+          const configFromApi = (await response.json()) as CLIConfigData;
+          this.$config = {
+            ...defaultConfig, // ensure the deploymentCdnUrl is set
+            ...configFromApi,
```

Comment:
Allows us to lean on local config for `deploymentCdnUrl` until the API catches up